src/modules/visitor-space/components/CreatedFilterForm/CreatedFilterForm.tsx

```diff
@@ -74,84 +84,160 @@ const CreatedFilterForm: FC<CreatedFilterFormProps> = ({ children, className, di
+			return dateRange;
+		}
+
+		if (form.operator === Operator.LessThanOrEqual) {
+			return endOfYear;
+		}
+
+		return startOfYear;
+	};
+
+	useEffect(() => {
+		if (year) {
+			const yearDate = convertYearToDate(year)?.toString();
+			setForm({ ...form, created: yearDate });
+		}
+	}, [year]);
+
+	useEffect(() => {
+		if (yearRange) {
+			setForm({ ...form, created: yearRange });
+		}
+	}, [yearRange]);
+
 	return (
 		<>
-			<div className={clsx(className, 'u-px-20 u-px-32:md')}>
-				<div className="u-mb-32">
-					<FormControl
-						className="u-mb-24 c-form-control--label-hidden"
-						errors={[errors.operator?.message]}
-						id={labelKeys.operator}
-						label={tHtml(
-							'modules/visitor-space/components/created-filter-form/created-filter-form___operator'
+			<div
+				className={clsx(
+					className,
+					styles['createdFilterForm__operatorSelect'],
+					'u-px-20 u-px-32:md'
+				)}
+			>
+				<FormControl
+					className={clsx('u-mb-24 c-form-control--label-hidden')}
+					errors={[errors.operator?.message]}
+					id={labelKeys.operator}
+					label={tHtml(
+						'modules/visitor-space/components/created-filter-form/created-filter-form___operator'
+					)}
+				>
+					<Controller
+						control={control}
+						name="operator"
+						render={({ field }) => (
+							<ReactSelect
+								{...field}
+								isDisabled={disabled}
+								components={{ IndicatorSeparator: () => null }}
+								inputId={labelKeys.operator}
+								onChange={(newValue) => {
+									const value = (newValue as SingleValue<SelectOption>)
+										?.value as Operator;
+
+									if (value !== form.operator) {
+										setForm({
+											operator: value,
+											created: defaultValues.created,
+										});
+									}
+								}}
+								options={operators}
+								value={getSelectValue(operators, field.value)}
+							/>
 						)}
-					>
-						<Controller
-							control={control}
-							name="operator"
-							render={({ field }) => (
-								<ReactSelect
-									{...field}
-									isDisabled={disabled}
-									components={{ IndicatorSeparator: () => null }}
-									inputId={labelKeys.operator}
-									onChange={(newValue) => {
-										const value = (newValue as SingleValue<SelectOption>)
-											?.value as Operator;
-
-										if (value !== form.operator) {
-											setForm({
-												operator: value,
-												created: defaultValues.created,
-											});
+					/>
+				</FormControl>
+			</div>
+			<div className={clsx(styles['createdFilterForm__date'], 'u-px-20 u-px-32:md')}>
+				<FormControl
+					className="u-mb-24 c-form-control--label-hidden"
+					errors={[errors.created?.message]}
+					id={labelKeys.created}
+					label={tHtml(
+						'modules/visitor-space/components/created-filter-form/created-filter-form___waarde'
+					)}
+				>
+					<SelectDateOrYear
+						yearsSelected={yearsSelected}
+						setYearsSelected={setYearsSelected}
+						showPluralLabel={showRange}
+					/>
```

Comment:
if we enter a year: 2023
and apply
then reopen the filter. It shows "1 Jan. 2023" with date selected instead of year.

Maybe we should default to the year radio box if the selected date starts with "1 Jan." when we open de filter?

![image](https://user-images.githubusercontent.com/1710840/214014098-e9a399f4-5509-4e79-81eb-bc38de7a6150.png)

Reply:
we'll create a separate jira task for this